mandatory/4-tax.js

```diff
@@ -5,7 +5,12 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+  function calculateSalesTax(price) {
```

Comment:
Indentation error! This line should start at the beginning!

Reply:
Thank you so much for feedback. I fixed it. :+1: